index.js

```diff
@@ -38,32 +40,55 @@ When your math is correct, monthlyRate will equal 1073.64
 If your name is `Oscar` mortgageCalculator() should return "Oscar, your monthly rate is 1073.64"
 */
 
-
-
-
+function mortgageCalculator(principal, interestRate, years) {
```

Comment:
you have years as a parameter but it's not being used. No need to take it in if it's not needed